src/containers/pages/PlanAssignment/index.tsx

```diff
@@ -72,6 +74,18 @@ interface PlanAssignmentProps extends JurisdictionTableProps {
   tree: TreeNode | null;
 }
 
+/**
+ * Check if map should be visible based on plan type
+ */
+export const isMapDisabled = (plan: PlanDefinition | null): boolean => {
+  let interventionType: string | undefined = '';
+  if (plan) {
+    const type = plan.useContext.find(element => element.code === INTERVENTION_TYPE_CODE);
+    type ? (interventionType = type.valueCodableConcept) : (interventionType = '');
+  }
+  return MAP_DISABLED_PLAN_TYPES.includes(interventionType);
+};
```

Comment:
This looks weird to me.  Questions:

- Do we have to account for plan being null here?

Suggestions:

- If plan is null just return false
- We may be able to avoid declaring `interventionType` as a variable